src/components/PivotTable/PivotTableValueCell.js

```diff
@@ -40,19 +45,32 @@ export const PivotTableValueCell = ({ row, column }) => {
         maxWidth: width,
     }
 
+    const cellRef = useRef(undefined)
```

Comment:
This needs to move above the early `return` on line 26

https://travis-ci.com/github/dhis2/analytics/builds/166674837#L4203

Reply:
Yes, I figured and fixed locally.